src/main/java/training/day4/homework/Vehicle.java

```diff
@@ -0,0 +1,58 @@
+package training.day4.homework;
+
+public class Vehicle {
+    private final static double DEFAULT_FUEL_CONSUMPTION = 1.25;
+    private double fuelConsumption;
+    private double fuel;
+    private int horsePower;
+
+    public Vehicle(double fuel, int horsePower) {
+        this.fuel = fuel;
+        this.horsePower = horsePower;
+        this.fuelConsumption = DEFAULT_FUEL_CONSUMPTION;
+    }
+
+    public void drive(double kilometers) {
+        double roadRun = (getFuel() / getFuelConsumption()) * 100;
```

Comment:
Do not need to use getters to access a field that is in the same class you are using it.
Instead, it can be done as so:
```java
double roadRun = (fuel / fuelConsumption) * 100;
```